### PR TITLE
implemented twilio sms features. Need to communicate with Sam to inte…

### PR DESCRIPTION
**Pull request by ddritzenhoff**
…grate with his code.

### CODE REVIEW

**Review by samlowe106 — APPROVED** [review]
LGTM

**Review by samlowe106 — APPROVED** [review]

File: klip/settings.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 """
 
 from pathlib import Path
+import os
```
Comment (mattognibene) [inline klip/settings.py:14]: is this import needed?

File: api/views.py [hunk 2]
```diff
@@ -179,4 +192,16 @@ def get_contacts(request):
...
     except Exception:
-        return JsonResponse({'error': 'Something went wrong'}, safe=False, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
\ No newline at end of file
+        return JsonResponse({'error': 'Something went wrong'}, safe=False, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+# Sends out the emergency message to user's emergency contacts.
+def _send_twilio_sms(message: str, userName, userContactNumbers):
+    # TODO: get user name and userContact numbers
+    message_to_broadcast = "Your friend " + userName + " needs assistance."
```
Comment (mattognibene) [inline api/views.py:201]: probably don't need this line; we can assume that the message passed in is already formatted with the user name

**Review by mattognibene — APPROVED** [review]
Minor feedback but overall looks good

**Pull request merged**